doc/articles/uno-publishing-maccatalyst.md

```diff
@@ -0,0 +1,53 @@
+---
+uid: uno.publishing.maccatalyst
+---
+
+# Publishing Your App for Mac Catalyst
+
+## Preparing For Publish
+
+- [Performance Profiling](xref:Uno.Tutorials.ProfilingApplications)
+
+## Building your app
+
+### Packaging your app using the MacCatalyst
```

Comment:
```suggestion
### Packaging your app using Mac Catalyst
```
Related to https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=126873&view=logs&jobId=050fb64b-9b12-52cc-b750-ec5d4e778b08&j=050fb64b-9b12-52cc-b750-ec5d4e778b08&t=669790bd-240a-5aea-d705-86a7326c36e7